local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  namespace: default
+  defaultMountDir: /local-disks
+configmap:
+  configMapName: "test"
+  storageClass:
+    - local-fast:
+        hostDir: "/mnt/fast"
+        mountDir: "/local-fast"
+    - local-normal:
```

Comment:
Let's remove this from the example. It's not adding much value

Reply:
done